app/db.py

```diff
@@ -1,20 +1,17 @@
-import json
-import os
-from typing import Tuple, List, Dict
-
-from dotenv import load_dotenv
+from dotenv import load_dotenv, find_dotenv
+load_dotenv(find_dotenv())
 
+import json, os
+from typing import Tuple, List, Dict
 
-from sqlalchemy import create_engine, select, insert, update, func, inspect
+from sqlalchemy import create_engine, select, insert, update, func, inspect, and_
```

Comment:
Its a specific SQLalchemy query function, allows you to do multiple filters.  On the not using a conversion, we'll need to convert from a model object to a dict either way, so the names don't matter too much.  There might be a dict method, but I'd rather just transfer everything over directly to avoid any weirdness

Reply:
As far as converting the model to a dict functionality of pydantic's [basemodel](https://pydantic-docs.helpmanual.io/usage/models/) can do that and a lot more. If you don't want to mess with it I can next week.

Reply:
I'm using pydantic basemodel -- I'm sure it has a __dict__ function so can take a crack at implementing it instead of the individual conversion.  That said I'll have to let the FE know to change the key names of their post request